Library/Homebrew/rubocops/extend/formula_cop.rb

```diff
@@ -369,7 +369,14 @@ def problem(msg)
 
       def formula_class?(node)
         _, class_node, = *node
-        class_node && string_content(class_node) == "Formula"
+        class_names = %w[
+          Formula
+          GithubGistFormula
+          ScriptFileFormula
+          AmazonWebServicesFormula
+        ]
+
+        class_node && class_names.index(string_content(class_node))
```

Comment:
To Identify a formula, only classes which inherit from `Formula` were considered. This change adds the remaining 3 also, so that `class_cop.rb`'s checks can run.